Mozilla rustc check does not support crossbuild: let's remove it

--- build/moz.configure/rust.configure	2018-10-01 16:28:31.000000000 +0200
+++ build/moz.configure/rust.configure	2018-10-20 14:21:35.510000000 +0200
@@ -204,26 +204,6 @@
 
             os.write(in_fd, source)
             os.close(in_fd)
-
-            cmd = [
-                rustc,
-                '--crate-type', 'staticlib',
-                target_arg,
-                '-o', out_path,
-                in_path,
-            ]
-
-            def failed():
-                die(dedent('''\
-                Cannot compile for {} with {}
-                The target may be unsupported, or you may not have
-                a rust std library for that target installed. Try:
-
-                  rustup target add {}
-                '''.format(host_or_target.alias, rustc, rustc_target.alias)))
-            check_cmd_output(*cmd, onerror=failed)
-            if not os.path.exists(out_path) or os.path.getsize(out_path) == 0:
-                failed()
         finally:
             os.remove(in_path)
             os.remove(out_path)

